cassovary-benchmarks/src/main/scala/com/twitter/cassovary/PageRankBenchmark.scala

```diff
@@ -24,4 +24,4 @@ class PageRankBenchmark(graph: DirectedGraph[Node],
   def operation() {
     PageRank(graph, pageRankParams)
   }
-}
\ No newline at end of file
+}
```

Comment:
can you report on the pagerank benchmark results before and after your change on a couple of graphs?